.github/workflows/test.yml

```diff
@@ -201,4 +199,5 @@ jobs:
         export PATH=${PATH}:`go env GOPATH`/bin
         export WEGO_BIN_PATH=$(pwd)/bin/wego
         go get -u github.com/onsi/ginkgo/ginkgo
```

Comment:
Curious why you replaced `go get -u github.com/onsi/ginkgo/ginkgo` with `go get github.com/onsi/ginkgo` above but not here :thinking: